admin_solutions/master_node_config.adoc

```diff
@@ -0,0 +1,401 @@
+method, then make your changes
+link:../admin_solutions/master_node_config.html#master-node-config-manual[manually in the configuration files] themselves.
+
+[[master-node-config-ansible]]
+=== Making Configuration Changes Using Ansible
+
+For this section, familiarity with Ansible is assumed. However, you can use this
+configuration as a reference to create your own implementation using the
+configuration management tool of your choosing (such as
+https://puppet.com/[Puppet], https://www.chef.io/[Chef],
+http://saltstack.com/[Salt]).
+
+Only a portion of the available options in the configuration files are
+https://github.com/openshift/openshift-ansible/blob/master/inventory/byo/hosts.ose.example[exposed to Ansible], so
+link:../install_config/install/advanced_install.html#configuring-cluster-variables[some options are configurable]
+with Ansible, but not all. After you install OpenShift, Ansible creates an
+inventory file with some values substituted in. Modifying this inventory allows
+you to customize your cluster by affecting changes in the master and node configuration files.
+
+The process of modifying configuration files using Ansible is similar
+to the installation process, because it uses the same playbook. When you modify
+the playbook and re-run it, Ansible carries out the required configuration changes.
+
+*Use Case: Configure the cluster to use HTPasswd authentication*
+
+[NOTE]
+====
+This use case assumes you have already set up SSH keys to all the nodes referenced in the playbook.
+====
+
+Initially, the OpenShift cluster would have been installed using a simple
+`ansible-playbook` command, which you will re-run once you've modified the
+inventory.
+
+To modify the Ansible inventory and make configuration changes:
+
+. Open the inventory file:
++
+.Sample inventory file:
+====
+
+$ atom ./hosts
+----
+[OSEv3:children]
+masters
+nodes
+
+[OSEv3:vars]
+ansible_ssh_user=cloud-user
+ansible_become=true
+#openshift_image_tag=v3.1.1.6
+#openshift_image_tag=v3.2.0.14
+deployment_type=openshift-enterprise
+
+openshift_docker_additional_registries=ec2-54-172-104-209.compute-1.amazonaws.com:5000
+openshift_docker_insecure_registries=ec2-54-172-104-209.compute-1.amazonaws.com:5000
+
+[masters]
+ec2-54-173-187-15.compute-1.amazonaws.com  openshift_ip=172.18.4.89 openshift_public_ip=54.173.187.15 openshift_hostname=ip-172-18-4-89.ec2.internal openshift_public_hostname=ec2-54-173-187-15.compute-1.amazonaws.com containerized=True
+[nodes]
+ec2-54-173-187-15.compute-1.amazonaws.com  openshift_ip=172.18.4.89 openshift_public_ip=54.173.187.15 openshift_hostname=ip-172-18-4-89.ec2.internal openshift_public_hostname=ec2-54-173-187-15.compute-1.amazonaws.com containerized=True openshift_schedulable=False
+ec2-54-89-5-36.compute-1.amazonaws.com  openshift_ip=172.18.4.90 openshift_public_ip=54.89.5.36 openshift_hostname=ip-172-18-4-90.ec2.internal openshift_public_hostname=ec2-54-89-5-36.compute-1.amazonaws.com containerized=True
```

Comment:
These were my AWS vm's and IPs at one point, I'm not sure if we're inadvertently linking subnet or other info we shouldn't be here. Is there any precedent in the docs for writing this file we could follow?